### PR TITLE
feat(grown-up-guide): UX & logic & navigation 

### PR DESCRIPTION
**Pull request by VladislavRiabko**
- add ButtonsMenu screen 
- add ButtonsMenu to Menu stack

### CODE REVIEW

File: app/navigation/stacks/menuStackNavigator.tsx [hunk 1]
```diff
@@ -3,14 +3,19 @@ import { createNativeStackNavigator } from 'react-native-screens/native-stack';
 
 import {
   MainMenuScreen,
-  ParentsGuideScreen,
   LanguageSettingsScreen,
+  ButtonsMenuScreen,
 } from '../../screens';
+import { IButtonsMenuItem } from '../../screens/Menu/ButtonsMenu/ButtonsMenu.data';
```
Comment (emberglaz) [inline app/navigation/stacks/menuStackNavigator.tsx:11]: import from screens

File: app/screens/Menu/ButtonsMenu/ButtonsMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import _ from 'lodash';
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { SafeAreaView, View } from 'react-native';
+import { IMAGES } from '../../../assets';
+
+import { ExtendedText, MainHeader } from '../../../components';
+import { generalStyles } from '../../../utils/styles';
+import { MenuButton } from '../components';
```
Comment (emberglaz) [inline app/screens/Menu/ButtonsMenu/ButtonsMenu.tsx]: refactor imports

File: app/screens/Menu/ButtonsMenu/ButtonsMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+export const ButtonsMenuScreen: React.FC<IButtonsMenuScreenProps> = ({
+  navigation,
+  route,
+}) => {
+  const { data, titleKey, subtitleKey } = route.params;
+
+  const { t } = useTranslation();
+
+  // TODO: change
+  const onMenuItemPress = useCallback((item: IButtonsMenuItem) => {}, []);
```
Comment (emberglaz) [inline app/screens/Menu/ButtonsMenu/ButtonsMenu.tsx]: use noop

File: app/screens/Menu/MainMenu/MainMenu.data.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { ImageSourcePropType } from 'react-native';
 import { IMAGES } from '../../../assets';
 
 export enum MainMenuItemType {
-  Guide = 'guide',
+  Guide = 'GrownUpGuide',
```
Comment (emberglaz) [inline app/screens/Menu/MainMenu/MainMenu.data.ts:7]: Guide = 'grownUpGuide',

File: app/screens/Menu/MainMenu/MainMenu.tsx [hunk 5]
```diff
@@ -9,18 +9,38 @@ import { useAppDispatch } from '../../../hooks';
...
+    (item: IMainMenuItem) => {
+      switch (item.type) {
+        case MainMenuItemType.Guide:
+          navigation.navigate('ButtonsMenu', {
+            data: GROWN_UP_GUIDE_ITEMS,
+            titleKey: 'screens.menu.grown_ups_guide.title',
+          });
+          break;
+        default:
+          // navigation.navigate(item.type);
```
Comment (emberglaz) [inline app/screens/Menu/MainMenu/MainMenu.tsx:39]: commented

File: app/screens/Menu/MainMenu/MainMenu.tsx [hunk 6]
```diff
@@ -9,18 +9,38 @@ import { useAppDispatch } from '../../../hooks';
...
 import { styles } from './MainMenu.styles';
 
-export const MainMenuScreen: React.FC<IMainMenuScreenProps> = () => {
+export const MainMenuScreen: React.FC<IMainMenuScreenProps> = ({
+  navigation,
+}) => {
   const { t } = useTranslation();
   const dispatch = useAppDispatch();
 
   // TODO: change when adding screens
```
Comment (emberglaz) [inline app/screens/Menu/MainMenu/MainMenu.tsx]: `// TODO: uncomment when adding screens`

**Review by emberglaz — CHANGES_REQUESTED** [review]

File: app/screens/Menu/MainMenu/MainMenu.data.ts [hunk 7]
```diff
@@ -2,11 +2,12 @@ import { ImageSourcePropType } from 'react-native';
 
 import { IMAGES } from '../../../assets';
 
+//* screens name
```
Comment (emberglaz) [inline app/screens/Menu/MainMenu/MainMenu.data.ts]: `//* This enum is used for navigation so names must be capitalized`

**Review by emberglaz — CHANGES_REQUESTED** [review]

**Review by Reykjavik151 — CHANGES_REQUESTED** [review]
But why did you delete ParentsGuide screen?

**Comment by VladislavRiabko** [comment]
> But why did you delete ParentsGuide screen?

We change naming and I remove it in this pr. But I save it in next pr.

**Review by Reykjavik151 — APPROVED** [review]

**Pull request merged**